File: timing.go
package main

import (
"fmt"
"time"
)

func main() {

	start := time.Now()
	x := 1.00001
	for i := 0; i < 100000; i++ {
		x = x * 1.00001
	}
	fmt.Println(x)
	t := time.Since(start)
	fmt.Println(t.Nanoseconds() / 1000.0)
}

File: timing.py
import datetime

def x():
	start = datetime.datetime.now()
	f = 1.00001
	for i in xrange(100000):
		f = f * 1.00001
	print f
	total_time = datetime.datetime.now() - start
	print total_time.total_seconds() * 10e6

x()
